### PR TITLE
AST-496 - fix: Added transition CSS to site-identity to make shrink sticky layout smoothen

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- When the Shrink Effect option enables
- Headers & Site identity creates a jerk

### Screenshots
- https://share.bsf.io/yAur2qgw
- Fix - https://share.bsf.io/GGuWRjvp

### Types of changes
- Bugfix

### How has this been tested?
- Enable Sticky Header along with Shrink effect
- Notice the header height jerk after a slight scroll

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 - Fix: Max-width responsive CSS enlarges logo size on mobile devices in old header layout.
 - Fix: Cart Icon not working with Sticky Header after enabling the option "Hide when scrolling down" on mobile devices.
 - Fix: Secondary menu not expendable on mobile when it is outside of Off Canvas.
-- Improvement: Added close button for multiselect selected items in Google font variants.
 - Fix: CLS – Flashing desktop menu on mobile view.
 - Fix: WooCommerce - product search widget button not aligned on other than shop and single product page.
-- Improvement: Changed customizer submenu control title from 'Container Animation' to 'Submenu Animation'.
-- Improvement: Moved 'Submenu Animation' control from General tab to Design  tab.
-- Improvement: Changed sub-menu container animation option name from 'Default' to 'None'.
+- Fix: Site Identity - Improved transition effect when sticky header & shrink options enable.
```
Comment (premanshup) [inline changelog.txt:12]:
```suggestion
- Fix: Sticky Header - Transition effect of headers when Shrink Effect is enable.
```

**Review by premanshup — CHANGES_REQUESTED** [review]

File: changelog.txt (hunk 1 above)
Comment (imnavanath) [reply changelog.txt]: Done 👍

File: changelog.txt [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 - Fix: Max-width responsive CSS enlarges logo size on mobile devices in old header layout.
 - Fix: Cart Icon not working with Sticky Header after enabling the option "Hide when scrolling down" on mobile devices.
 - Fix: Secondary menu not expendable on mobile when it is outside of Off Canvas.
-- Improvement: Added close button for multiselect selected items in Google font variants.
 - Fix: CLS – Flashing desktop menu on mobile view.
 - Fix: WooCommerce - product search widget button not aligned on other than shop and single product page.
-- Improvement: Changed customizer submenu control title from 'Container Animation' to 'Submenu Animation'.
-- Improvement: Moved 'Submenu Animation' control from General tab to Design  tab.
-- Improvement: Changed sub-menu container animation option name from 'Default' to 'None'.
+- Fix: Sticky Header - Transition effect of headers when Shrink Effect is enable.
```
Comment (sagarb90) [inline changelog.txt:12]: Fix: Sticky Header - Transition effect of headers when Shrink Effect is enabled.

**Review by abhishek-bsf — APPROVED** [review]

File: changelog.txt (hunk 2 above)
Comment (imnavanath) [reply changelog.txt]: Done 👍

**Pull request merged**